SwitchSpeak-iOS/SwitchSpeak-iOS/ViewController.swift

```diff
@@ -37,5 +37,13 @@ class ViewController: UIViewController {
 		super.didReceiveMemoryWarning()
 		// Dispose of any resources that can be recreated.
 	}
+    
+    // Control + Command + z = Shake on the device simulator.
+    // For now we can use this to make the iPad speak. Later we will need a button.
+    override func motionEnded(_ motion: UIEventSubtype, with event: UIEvent?) {
```

Comment:
don't know how to test this function on the simulator. The code seems fine to me.